pkg/upload/js.go

```diff
@@ -222,8 +221,9 @@ func addSourcesContent(section map[string]interface{}, sourceMapPath string, pro
 				sourcePath = sourcePath[:questionMark-1]
 			}
 		}
-		if !path.IsAbs(sourcePath) {
-			sourcePath = path.Join(projectRoot, sourcePath)
```

Comment:
I think you can remove `projectRoot` from param list and further up the call chain?